nunit.sln

```diff
@@ -312,55 +315,55 @@ Global
 		{B0791F6D-5D14-40DD-A2CB-5D2A97892693} = {64FE69B2-CE0B-4A47-83A3-9DE803851447}
 		{EA9871E3-9972-4476-821E-3961C28125E8} = {64FE69B2-CE0B-4A47-83A3-9DE803851447}
-		{28C7B9D6-BE1F-45E5-952C-0D5C9CA3EFDF} = {64FE69B2-CE0B-4A47-83A3-9DE803851447}
+		{83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3} = {5D8A9D62-C11C-45B2-8965-43DE8160B558}
 		{6A281C98-B74D-403B-8536-966871B992E3} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
 		{DDCE1AE3-AA0D-4E2D-AD1B-8CA322AFA7DD} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
 		{1D38AEC8-F5C0-4B1C-A0AE-18F4015A2E78} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
 		{A35E6AD6-6D46-41E1-8DB1-A41086793893} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
 		{08D0F53A-CD37-4060-B5A2-BCC202F0DE46} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
-		{DBD0C8B0-BE4E-4CBF-AAD4-44BEA2EA830B} = {83154E24-B0DD-48E7-8BB5-55CF3D7B0FE3}
+		{7B573D52-181D-482F-A61C-82E543B3E958} = {5D8A9D62-C11C-45B2-8965-43DE8160B558}
 		{D209C368-1277-4EA6-A887-AA6EBA51AB99} = {7B573D52-181D-482F-A61C-82E543B3E958}
 		{C3264FF3-E2D7-48D7-93A7-CDC9A827AD9D} = {7B573D52-181D-482F-A61C-82E543B3E958}
 		{705FBD41-77A9-4FE7-A6C1-85026F2DDA09} = {7B573D52-181D-482F-A61C-82E543B3E958}
 		{E7541163-D7BB-4E9A-A722-D7737B7A2995} = {7B573D52-181D-482F-A61C-82E543B3E958}
 		{5B8B0EAB-3E74-48BB-8231-26E73B2019DC} = {7B573D52-181D-482F-A61C-82E543B3E958}
-		{46EC0BC1-8F51-4B4D-B967-2EE452EA68FC} = {7B573D52-181D-482F-A61C-82E543B3E958}
-		{3DEB15F9-E7DA-403F-B6D3-A8499310397F} = {A21BFC37-44FC-41A0-B6E7-6F949F45A635}
-		{A76794DA-29A4-4D63-9850-E0E363A2FAFD} = {A21BFC37-44FC-41A0-B6E7-6F949F45A635}
-		{0A5F920A-1BF5-4DAC-B799-0C618B203797} = {A21BFC37-44FC-41A0-B6E7-6F949F45A635}
-		{A4F0C741-6ED4-46C8-AE75-57E1C7197E22} = {A21BFC37-44FC-41A0-B6E7-6F949F45A635}
+		{A21BFC37-44FC-41A0-B6E7-6F949F45A635} = {5D8A9D62-C11C-45B2-8965-43DE8160B558}
 		{A972031D-2F61-4183-AF75-99EE1A9F6B32} = {49D441DF-39FD-4F4D-AECA-86CF8EFE23AF}
 		{775FAD50-3623-4922-97C4-DFB29A8BE4C7} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
 		{372A3447-D657-40FF-A089-77C19FEC30C8} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
 		{D694CB69-6CFB-4762-86C2-EB27B808B282} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
 		{C2A8FC7A-FA64-46EA-AF6D-73D6B371DBF8} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{28B605B2-E2E9-417E-8369-49E263F1F31B} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{91DD5A2D-9FE3-4C3C-9253-876141874DAD} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{B923C1E7-54DD-4E79-A1A9-B21123863B04} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{CF9861F6-270B-4C90-9264-97F33E2DF707} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{E2D74AD3-BF60-435F-B109-D8420036DDC7} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
-		{1BCC40A8-C3E2-4852-9699-4D18F0C8944C} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
 		{0DE218CA-AFB8-423A-9CD2-E22DEAC55C46} = {576DB1E6-C5EC-4FEF-A826-EC19D8BEE572}
 		{B310A760-8AE1-41CA-81F8-03B12E2FCE30} = {576DB1E6-C5EC-4FEF-A826-EC19D8BEE572}
+		{28B605B2-E2E9-417E-8369-49E263F1F31B} = {31B45C4C-206F-4F31-9CC6-33BF11DFEE39}
```

Comment:
Add one file to the solution and Visual Studio makes all these changes to the solution file. It's no wonder we can never merge the solution file :-1: